2021-02-17 19:54:03,053 INFO    MainThread:239640 [wandb_setup.py:_flush():70] setting env: {}
2021-02-17 19:54:03,053 INFO    MainThread:239640 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-02-17 19:54:03,054 INFO    MainThread:239640 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-17 19:54:03,054 INFO    MainThread:239640 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-17 19:54:03,054 INFO    MainThread:239640 [wandb_init.py:_log_setup():303] Logging user logs to /home/tyna/Documents/safe-experts/algos/wandb/run-20210217_195403-23fogmeq/logs/debug.log
2021-02-17 19:54:03,054 INFO    MainThread:239640 [wandb_init.py:_log_setup():304] Logging internal logs to /home/tyna/Documents/safe-experts/algos/wandb/run-20210217_195403-23fogmeq/logs/debug-internal.log
2021-02-17 19:54:03,993 INFO    MainThread:239640 [wandb_run.py:_console_start():1237] atexit reg
2021-02-17 19:54:03,994 INFO    MainThread:239640 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.REDIRECT
2021-02-17 19:54:03,994 INFO    MainThread:239640 [wandb_run.py:_redirect():1107] Redirecting console.
2021-02-17 19:54:03,994 INFO    MainThread:239640 [redirect.py:install():204] install start
2021-02-17 19:54:03,995 INFO    MainThread:239640 [redirect.py:install():219] install stop
2021-02-17 19:54:03,995 INFO    MainThread:239640 [redirect.py:install():204] install start
2021-02-17 19:54:03,995 INFO    MainThread:239640 [redirect.py:install():219] install stop
2021-02-17 19:54:03,995 INFO    MainThread:239640 [wandb_run.py:_redirect():1151] Redirects installed.
2021-02-17 20:08:14,221 INFO    MainThread:239640 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 255
2021-02-17 20:08:14,222 INFO    MainThread:239640 [wandb_run.py:_restore():1179] restore
2021-02-17 20:08:14,222 INFO    MainThread:239640 [redirect.py:uninstall():223] uninstall start
2021-02-17 20:08:14,222 INFO    MainThread:239640 [redirect.py:_stop():278] _stop: stdout
2021-02-17 20:08:14,222 INFO    MainThread:239640 [redirect.py:_stop():284] _stop closed: stdout
2021-02-17 20:08:14,222 INFO    stdout    :239640 [redirect.py:_pipe_relay():120] relay done saw last write: stdout
2021-02-17 20:08:14,222 INFO    stdout    :239640 [redirect.py:_pipe_relay():136] relay done done: stdout
2021-02-17 20:08:14,222 INFO    MainThread:239640 [redirect.py:_stop():290] _stop joined: stdout
2021-02-17 20:08:14,222 INFO    MainThread:239640 [redirect.py:_stop():292] _stop rd closed: stdout
2021-02-17 20:08:14,222 INFO    MainThread:239640 [redirect.py:uninstall():227] uninstall done
2021-02-17 20:08:14,222 INFO    MainThread:239640 [redirect.py:uninstall():223] uninstall start
2021-02-17 20:08:14,222 INFO    MainThread:239640 [redirect.py:_stop():278] _stop: stderr
2021-02-17 20:08:14,223 INFO    MainThread:239640 [redirect.py:_stop():284] _stop closed: stderr
2021-02-17 20:08:14,223 INFO    stderr    :239640 [redirect.py:_pipe_relay():120] relay done saw last write: stderr
2021-02-17 20:08:14,223 INFO    stderr    :239640 [redirect.py:_pipe_relay():136] relay done done: stderr
2021-02-17 20:08:14,223 INFO    MainThread:239640 [redirect.py:_stop():290] _stop joined: stderr
2021-02-17 20:08:14,223 INFO    MainThread:239640 [redirect.py:_stop():292] _stop rd closed: stderr
2021-02-17 20:08:14,223 INFO    MainThread:239640 [redirect.py:uninstall():227] uninstall done
2021-02-17 20:08:14,229 INFO    MainThread:239640 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "da37ed5a935641ab816bdee717c80816"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 747
      total_bytes: 747
    }
  }
}

2021-02-17 20:08:16,232 INFO    MainThread:239640 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "672e6e0f67574f5eb3da9941200ba94d"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 364169
      total_bytes: 904166
    }
  }
}

2021-02-17 20:08:18,236 INFO    MainThread:239640 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "c2d8e91964e84091a666ce4df1670fa8"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 904166
      total_bytes: 904166
    }
  }
}

2021-02-17 20:08:19,347 INFO    MainThread:239640 [wandb_run.py:_show_summary():1469] rendering summary
2021-02-17 20:08:19,347 INFO    MainThread:239640 [wandb_run.py:_show_history():1507] rendering history
2021-02-17 20:08:19,349 INFO    MainThread:239640 [wandb_run.py:_show_files():1536] logging synced files
2021-02-17 20:08:19,350 INFO    MainThread:239640 [internal.py:handle_exit():138] Internal process exited
